### PR TITLE
Hw24

### PR DESCRIPTION
**Pull request by Engelko**

### CODE REVIEW

File: Bash_scripts/HW24/from_hw23/pass1 [hunk 1]
```diff
@@ -0,0 +1 @@
+tutpassword
```
Comment (mavrik90) [inline Bash_scripts/HW24/from_hw23/pass1:1]: I hope this is not real password? ;)

File: Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+ # Start Services Docker,Nginx
+ - name: "Docker started"
+   ansible.builtin.service:
+     name: "{{ docker }}"
```
Comment (mavrik90) [inline Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml]: No sense to use variables here and below

File: Bash_scripts/HW24/from_hw23/pass1 (hunk 1 above)
Comment (Engelko) [reply Bash_scripts/HW24/from_hw23/pass1:1]: Example

File: Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml (hunk 2 above)
Comment (Engelko) [reply Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml]: I agree,it's too much

Comment (Engelko) [reply Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml]: fixed

File: Bash_scripts/HW24/run_ansible.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+if [ "$SORTIDA" -eq 0 ];
+then
+    echo "Updated!"
+else
+    echo "Update failed!"
+fi
+
+echo "Ending log" >> $LOGFILE
+
+exit $SORTIDA
```
Comment (mavrik90) [inline Bash_scripts/HW24/run_ansible.sh]: new line

File: Bash_scripts/HW24/run_ansible.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+# echo -n "Uncompressing update... " >> $LOGFILE
+# cd $DESTINATION_DIR
+# unzip $PLAYBOOK_GZ
+# rm $PLAYBOOK_GZ
+# SORTIDA=$?
+# echo OK >> $LOGFILE
+
+echo "Start Ansible... " >> $LOGFILE
+cd $DESTINATION_DIR/$WORK_FOLDER
+ansible-playbook $PLAYBOOK_FILE -i $INVENTORY_FILE -i $INVENTORY_FILE_2 >> $LOGFILE
```
Comment (mavrik90) [inline Bash_scripts/HW24/run_ansible.sh]: This will be broken for only one one inventory file

File: Bash_scripts/HW24/run_ansible.sh (hunk 3 above)
Comment (Engelko) [reply Bash_scripts/HW24/run_ansible.sh]: fixed

File: Bash_scripts/HW24/run_ansible.sh (hunk 4 above)
Comment (Engelko) [reply Bash_scripts/HW24/run_ansible.sh]: Yes, I fixed it. I changed file hosts.yml and chenged run_ansible.sh
hw26t20

File: Bash_scripts/HW24/from_hw23/pass1 (hunk 1 above)
Comment (mavrik90) [reply Bash_scripts/HW24/from_hw23/pass1:1]: Please resolve this

File: Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml (hunk 2 above)
Comment (mavrik90) [reply Bash_scripts/HW24/from_hw23/roles/hw23/handlers/main.yml]: please resolve it

**Review by mavrik90 — APPROVED** [review]
Well done!

**Pull request merged**